spec/features/users/new_spec.rb

```diff
@@ -22,6 +20,28 @@
     expect(user.email).to eq('psue@turing.edu')
   end
 
-  xit 'denies a new user if the email is not unique' do
+  it 'denies a new user if the email is not unique' do
+    create(:user, name: 'Alfred', email: 'psue@turing.edu')
+
+    visit '/register'
+
+    fill_in('name', with: 'Peggy Sue')
+    fill_in('email', with: 'psue@turing.edu')
+
+    click_button('Register')
+
+    expect(page).to have_content('Email has already been taken')
+  end
+
+  it 'denies a new user if any field is blank' do
+    visit '/register'
+
+    fill_in('name', with: '')
+    fill_in('email', with: '')
+
+    click_button('Register')
+
+    expect(page).to have_content("Name can't be blank")
+    expect(page).to have_content("Email can't be blank")
```

Comment:
Thanks for getting these sad paths and edgecases built in to make our test more robust!